bindings/tester/precompiles.go

```diff
@@ -202,6 +214,10 @@ func CallPrecompiledContracts(address int, lt *LoadTester, opts *bind.TransactOp
 		log.Trace().Str("method", "TestBlake2f").Msg("Executing contract method")
 		inputData = GenerateECPairingInput()
 		return lt.TestBlake2f(opts, inputData)
+	case 100:
```

Comment:
100 or 0x100=256?

Reply:
I'd go for `0x100` but make sure to modify all the other to follow the same standard. Same thing for https://github.com/0xPolygon/polygon-cli/blob/main/bindings/tester/precompiles.go#L210

Reply:
Well I took a second look at it and I think `100` would be better in the end.

Since we're used to the following:

```bash
polycli loadtest --mode px --function 3
polycli loadtest --mode px --function 9
// etc.
```

I feel like this call:

```bash
polycli loadtest --mode px --function 100
```

makes more sense than this call:

```bash
polycli loadtest --mode px --function 256
```

Reply:
Also wrong call from my end, I prefer we remove this precompile from `GetRandomPrecompiledContractAddress` since it's not supported on Ethereum mainnet.

Reply:
Finished, pls merge @leovct